docs/notebooks/util/plotting_plotly.py

```diff
@@ -160,6 +161,47 @@ def plot_gp_plotly(
     return fig
 
 
+def plot_de_plotly(
```

Comment:
Why do you need a specific function for DE? you should be able to use the gp one?

If not, docstring needs updating

Reply:
gp model one uses predict_f method, that's why...

while we are at it, this whole plotting facility needs a rehaul, its poorly documented and can be made more reusable between the models - we could make it its own package? not a high priority thing, but nice visualization tools would be a plus for users...

Reply:
ok, I have refactored these particular functions now, this triggered some changes in the notebooks, you can probably ignore those